front-end/src/app/shared/models/transaction-groups/transaction-group-aa.model.ts

```diff
@@ -103,7 +103,7 @@ export class TransactionGroupAA extends DoubleTransactionGroup {
 
   getAutoGeneratedChildFields(childTemplateMap: TransactionTemplateMapType): string[] {
     return this.getChildFormProperties(childTemplateMap).filter(
-      (field) => ![childTemplateMap.text4000].includes(field)
+      (field) => ![childTemplateMap.text4000, childTemplateMap.category_code].includes(field)
```

Comment:
Did you mean to add memo_code rather than category_code?

Reply:
<img width="1169" alt="image" src="https://github.com/fecgov/fecfile-web-app/assets/97105825/c7637fa4-d87c-4f4d-8c2d-50bdfe0b79c8">

I think the user is allowed to edit that part.  The schema shows it as variable, like the memo

Reply:
If it's not editable, should it be in the interface/schema at all?  it will never be populated

Reply:
I just figured out what i'm saying and what you did after testing it locally 🤦